src/index.html

```diff
@@ -1,13 +1,275 @@
+          </article>
+
+          <article class="products__item item">
+            <img
+              class="item__photo image"
+              src="images/Ride_in_town_ST.png"
+              alt="Ride in town ST"
+            >
+            <h3 class="item__title">Ride in town ST</h3>
+            <p class="item__about">
+              An open frame for an upright riding position as the most comfortable ride in town.
+            </p>
+            <p class="item__price">$ 2 590</p>
+          </article>
+
+          <article class="products__item item">
+            <img
+              class="item__photo image"
+              src="images/Agile_ride_3.png"
+              alt="Agile ride 3"
+            >
+            <h3 class="item__title">Agile ride 3</h3>
+            <p class="item__about">
+              The lightweight frame that has earned its street tread as a sleek, agile ride.
+            </p>
+            <p class="item__price">$ 2 090</p>
+          </article>
+        </div>
+      </section>
+
+      <section class="details" id="details">
+        <h2 class="section-title">The details</h2>
+        <div class="detail-block">
+          <img
+            class="detail-block__image-left image"
+            src="./images/Auto Unlock_1.png"
+            alt="Auto Unlock"
+          >
+          <img
+            class="detail-block__image-right image"
+            src="./images/Auto Unlock_2.png"
+            alt="Auto Unlock"
+            >
+          <h3 class="detail-block__title">Auto Unlock</h3>
+          <p class="detail-block__content">
+            The app senses when you're nearby to unlock automatically.
+            GPS tracking so you know where your bike is and can track it anytime.
+          </p>
+        </div>
+
+        <div class="detail-block">
+          <img
+            class="detail-block__image-left image"
+            src="./images/Range & Integrated_1.png"
+            alt="Range & Integrated lights"
+          >
+          <img
+            class="detail-block__image-right image"
+            src="./images/Range & Integrated_2.png"
+            alt="Range & Integrated lights"
+            >
+          <h3 class="detail-block__title">Range & Integrated lights</h3>
+          <p class="detail-block__content">
+            The removable battery has up to 70km battery autonomy and weighs only 2.4 kg.
+            Lights integrated into the frame give you always-on visibility day and night.
+          </p>
+        </div>
+
+        <div class="detail-block">
+          <img
+            class="detail-block__image-left detail-block__image-left--minor image"
+            src="./images/Hydraulic disc_1.png"
+            alt="Hydraulic disc brakes & Lightweight"
+          >
+          <img
+            class="detail-block__image-right detail-block__image-right--main image"
+            src="./images/Hydraulic disc_2.png"
+            alt="Hydraulic disc brakes & Lightweight"
```

Comment:
Nice description, but now the screenreader will read these two images as the same because the descriptions are identical.